services/frontend/src/pages/HeroDetailPage.tsx

```
@@ -0,0 +1,243 @@
+  return (
+    <div>
+      <p className="list-label">{label}</p>
+      <ol className="matchup-list">
+        {rows.map(r => {
+          const opponentId = r.hero_id === heroId ? r.opponent_id : r.hero_id
+          return (
+            <li
+              key={opponentId}
+              className="matchup-row"
+              onClick={() => navigate(`/heroes/${opponentId}/1`)}
+            >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)